lms/djangoapps/grades/transformer.py

```diff
@@ -49,6 +50,12 @@ def collect(cls, block_structure):
         """
         block_structure.request_xblock_fields(*cls.FIELDS_TO_COLLECT)
         cls._collect_max_scores(block_structure)
+        collect_unioned_set_field(
+            block_structure=block_structure,
+            transformer=cls,
+            merged_field_name='containing_subsections',
```

Comment:
`inclusive_subsections`?
`scoring_affected_subsections`?

Taking a step back, can we save this field only on scorable blocks?  In which case, it wouldn't matter, because subsections aren't scorable.